### PR TITLE
feat: add gitpod help doc link

### LINKED ISSUE #57296: Challenges with Gitpod starter projects should link to Gitpod help documentation
### Describe the Issue

I know we added a link to the forum post at the top of the [backend section](https://www.freecodecamp.org/learn/back-end-development-and-apis/), but it isn't working, and it's impossible to help people using Gitpod because we can never see their code. We keep having to tell people how to share their code.

Any challenge that has a Gitpod starter project should include a link to the documentation about how to get help.

https://forum.freecodecamp.org/t/how-to-use-gitpod-in-the-curriculum/668669#how-can-i-share-my-workspace-to-get-help-8

### Affected Pages

Back End Development and APIs

https://www.freecodecamp.org/learn/back-end-development-and-apis/managing-packages-with-npm/how-to-use-package-json-the-core-of-any-node-js-project-or-npm-package
https://www.freecodecamp.org/learn/back-end-development-and-apis/basic-node-and-express/meet-the-node-console
https://www.freecodecamp.org/learn/back-end-development-and-apis/mongodb-and-mongoose/install-and-set-up-mongoose
https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/timestamp-microservice
https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/request-header-parser-microservice
https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/url-shortener-microservice
https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker
https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/file-metadata-microservice

Quality Assurance

https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-and-testing-with-chai/learn-how-javascript-assertions-work
https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/set-up-a-template-engine
https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/metric-imperial-converter
https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/issue-tracker
https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/personal-library
https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/sudoku-solver
https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/american-british-translator

Information Security

https://www.freecodecamp.org/learn/information-security/information-security-with-helmetjs/install-and-require-helmet
https://www.freecodecamp.org/learn/information-security/information-security-projects/stock-price-checker
https://www.freecodecamp.org/learn/information-security/information-security-projects/anonymous-message-board
https://www.freecodecamp.org/learn/information-security/information-security-projects/port-scanner
https://www.freecodecamp.org/learn/information-security/information-security-projects/sha-1-password-cracker
https://www.freecodecamp.org/learn/information-security/information-security-projects/secure-real-time-multiplayer-game

### Your code

```



```

### Expected behavior

Each challenge should include instruction about how to get help with Gitpod code.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by carlosplanchart**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #57296 

Per https://github.com/freeCodeCamp/freeCodeCamp/issues/57296#issuecomment-2501511754, the following pages from these topics also include a link to the documentation:
- Information Security with HelmetJS
- Information Security Projects
- Quality Assurance
- Quality Assurance Projects

### CODE REVIEW

File: curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ dashedName: exercise-tracker
...
 
 -   Clone <a href="https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">this GitHub repo</a> and complete your project locally.
--   Use <a href="https://gitpod.io/?autostart=true#https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">our Gitpod starter project</a> to complete your project.
+-   Use <a href="https://gitpod.io/?autostart=true#https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">our Gitpod starter project</a> to complete your project. Learn how to share your Gitpod workspace before asking for help <a href="https://forum.freecodecamp.org/t/how-to-use-gitpod-in-the-curriculum/668669#how-can-i-share-my-workspace-to-get-help-8" target="_blank" rel="noopener noreferrer nofollow">here</a>.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md:14]: "here" is not an accessible anchor text, please make the anchor text describe what it links to

File: curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ dashedName: exercise-tracker
...
 
 -   Clone <a href="https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">this GitHub repo</a> and complete your project locally.
--   Use <a href="https://gitpod.io/?autostart=true#https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">our Gitpod starter project</a> to complete your project. Learn how to share your Gitpod workspace before asking for help <a href="https://forum.freecodecamp.org/t/how-to-use-gitpod-in-the-curriculum/668669#how-can-i-share-my-workspace-to-get-help-8" target="_blank" rel="noopener noreferrer nofollow">here</a>.
+-   Use <a href="https://gitpod.io/?autostart=true#https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">our Gitpod starter project</a> to complete your project. Learn how to <a href="https://forum.freecodecamp.org/t/how-to-use-gitpod-in-the-curriculum/668669#how-can-i-share-my-workspace-to-get-help-8" target="_blank" rel="noopener noreferrer nofollow">share your Gitpod workspace to get help</a>.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md:14]:
```suggestion
-   Use <a href="https://gitpod.io/?autostart=true#https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/" target="_blank" rel="noopener noreferrer nofollow">our Gitpod starter project</a> to complete your project. Learn <a href="https://forum.freecodecamp.org/t/how-to-use-gitpod-in-the-curriculum/668669#how-can-i-share-my-workspace-to-get-help-8" target="_blank" rel="noopener noreferrer nofollow">how to share your Gitpod workspace to get help</a>.
```

this is more clear, remember that the anchor text need to make sense on its own

Comment (cplan85) [reply curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md]: Thank you for your feedback. I have updated the link in all the associated files per your suggestion.

**Review by Sembauke — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]

**Comment by ilenia-magoni** [comment]
Hi @cplan85 . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**